debian/changelog

```diff
@@ -1,9 +1,27 @@
-frr (9.1~dev-1) UNRELEASED; urgency=medium
+frr (9.2~dev-1) UNRELEASED; urgency=medium
 
-  * FRR Dev 9.1
+  * FRR Dev 9.2
+
+ -- Donatas Abraitis <donatas@opensourcerouting.org>  Tue, 10 Oct 2023 12:00:00 -0600
+
+frr (9.1-0) UNRELEASED; urgency=medium
+
+  * New upstream release FRR 9.1
 
  -- Jafar Al-Gharaibeh <jafar@atcorp.com>  Tue, 06 Jun 2023 12:00:00 -0600
 
+frr (9.0.1-0) unstable; urgency=medium
+
+  * New upstream release FRR 9.0.1
+
+ -- Jafar Al-Gharaibeh <jafar@atcorp.com>  Fri, 01 Sep 2023 02:00:00 -0600
+
```

Comment:
We typically keep the bug fix entries on the the stables branches and don't bring then to master. Can you please drop this?

Reply:
Thanks, done.